### DISCUSSION TITLE
Is there a Operation Column

### DISCUSSION THREAD

**Discussion by vuslatx** [winsiderss/systeminformer#1127] [Questions / Answers] [answered]
First, I love processhacker too much. Thanks to developers.

I want to see what changes the programs made. Especially, regedit changes. 

This is how it is in process monitor. 

![image](https://user-images.githubusercontent.com/65629417/154021423-bacaaef0-8373-471c-8a59-1869bd6ed59b.png)

Is there a way to do this? Thanks..

**Comment by dmex** [comment, marked as answer]
> Is there a way to do this? 

This isn't supported at present.
